src/components/ec-smart-table/ec-smart-table.vue

```diff
@@ -199,6 +208,9 @@ const isEmpty = computed(() => (props.data ?? []).length === 0);
 
 // slots
 const slots = useSlots();
+const isInCustomSlotThreshold = useMediaQuery('(max-width: 768px)');
```

Comment:
This functionality is already implemented in `ec-table` here we should just let user of smart table possibility to force using the `row` slot.